nomenclature/definitions/variable/technology/power-plant.yaml

```
@@ -1,4 +1,135 @@
-# description of thermal power plants
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term cost   [€/h]   = Constant Var Cost [Mcal/h] * Price|Primary Energy [€/Mcal]
+Constant Var Cost|Electricity|Coal:
```

suchest to rename that to
```suggestion
Operational Cost|Constant Term|Electricity Generation|Coal:
```